docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,115 @@
+// This is a parameterized module. Parameters used:
+//
+// parameter-jar-file-name: runtime-specific default target Maven build artifact name.
+// parameter-version-runtime-plugin: runtime-specific name for the runtime version property variable in the POM file.
+// parameter-artifact-name-runtime-plugin: runtime plugin name in the POM file.
+// parameter-group-name-runtime-plugin: runtime plugin Maven artifact group name.
+// parameter-runtime-name: Runtime specific atribute for substituting the approrpiate expanded rumtime name.
+// parameter-link-appdev-guide: link to the procedure providing a complete example of the appropriate runtime-specific POM file configuration.
+//  context: used in anchor IDs to conflicts due to duplicate IDs.
+//
+// Rationale: This procedure is the same for 2 or more runtimes.
+[id='deploying-an-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+Deploy your {parameter-runtime-name} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A Maven-based {parameter-runtime-name} application.
+* The Red Hat Maven repository link:{parameter-link-appdev-guide}[specified] in the `pom.xml` of your project.
+* RHEL 7 installed.
+* OpenJDK 8 installed.
+* Maven 3.5 or later installed.
+
+
+.Procedure
+
+. Navigate to the directory with your application project:
++
+[source,bash,subs="attributes+",options="nowrap"]
+----
+$ cd myApp
+----
+
+Ensure that you properly configure the {parameter-runtime-name} Maven plugin:
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<project ... >
```

Comment:
For some reason in rendered version I am seeing `+`: https://deploy-preview-1256--launcher-documentation.netlify.com/docs/spring-boot-runtime.html#creating-a-basic-springboot-application_spring-boot, in section `5.1.2` after `Ensure that you properly configure the Spring Boot Maven plugin:`

Reply:
@geoand Thanks for the catch. My bad, as I deleted a line form the procedure and did not add the numbering back. Fixed it now.

Reply:
Thanks!